Jenkinsfile

```diff
@@ -133,6 +133,8 @@ if (env.CHANGE_TITLE && env.CHANGE_TITLE.contains('[ci-skip]')) {
                                             sh "npm run vendor-install"
                                             sh "unzip ${env.KEYBASE_JS_VENDOR_DIR}/flow/flow-linux64*.zip -d ${env.BASEDIR}"
                                             sh "${env.BASEDIR}/flow/flow status shared"
+                                            sh "npm run shrinkwrap-audit"
```

Comment:
The script is dependent on the `babel-node` package (and related helpers) that are installed in `desktop/` by `npm run vendor-install`, I believe.